process/phases.md

```diff
@@ -45,22 +45,22 @@ Entry requirements:
 During this phase:
 
   1. If the proposal is not already listed, it should be added to the [proposal list](https://github.com/WebAssembly/proposals/blob/main/README.md) at this time.
-  1. A new repository, forking the spec repo, is created by one of the WebAssembly organization administrators, or transfered to the WebAssembly organization by the champion.
+  1. A new repository, forking the spec repo, is created by one of the WebAssembly organization administrators, or transfered to the WebAssembly organization by the champion. See the [how-to](https://github.com/WebAssembly/proposals/blob/main/howto.md) for this step.
   1. The champion will attempt to reach broad consensus in the Community Group.
-  1. Pull requests and issues are used to iterate on the design of the feature. Specifically, the spec text must be in a reasonably complete state before attempting to move to phase 2.
+  1. Pull requests and issues are used to iterate on the design of the feature. Specifically, an overview document must be produced that specifies the feature with reasonably precise and complete language before attempting to move to phase 2 (meaning it is sufficiently precise to be implemented following this description, without obvious holes or ambiguities).
   1. If relevant to demonstrate the viability of a feature, prototype implementations of the feature are implemented by interested embedders (possibly on a branch).
 
-## 2. Proposed Spec Text Available [Community + Working Group]
+## 2. Feature Description Available [Community + Working Group]
 
 Entry requirements:
 
-   * Full proposed English spec text available in a forked repo around which a reasonably high level of consensus exists.
-   * *Updates to the formal notation, test suite, and reference interpreter are NOT yet required.*
+   * Precise and complete overview document is available in a forked repo around which a reasonably high level of consensus exists.
+   * *Updates to the actual spec document, test suite, and reference interpreter are NOT yet required.*
 
 During this phase:
 
    * One or more implementations proceed on prototyping the feature to the point that a comprehensive set of tests can be added.
-   * A test suite is added. These tests need not pass the reference interpreter at this point, but should pass on some implementation.
+   * A test suite is added in the forked repo. These tests need not pass the reference interpreter at this point, but should pass on *some* implementation.
```

Comment:
Should *some* be qualified? Historically the tests at this stage do pass on a prototype implementation in one of the production engines.

Reply:
This is primarily a sanity check that the written tests make sense, not a validation of the implementation. For most proposals I worked on, it actually was the reference interpreter that supplied this, since engine implementations only happened much later. We explicitly do not require work on engines at this stage yet (that's listed in the next phase).

Reply:
I guess I've always found this requirement somewhat confusing, how do we get a good signal for a sanity check if they don't run against the interpreter or some sort of representative implementation? I think this should go hand in hand with either the reference implementation, or an implementation in some representative engine (even if that means that we move this to a different phase). That said I realize that this concern may be out of scope for this particular PR as it's improving documentation, so this is not a blocking comment, more of raising a concern as I've read this document in detail after some time.

Reply:
Well, at this stage only a prototype is required, in no specific form. So the only reasonable requirement is to run tests against that. We could tighten the requirements about the form that such a prototype takes, but I'm not convinced we should do that.

However, I added the requirement that engine implementations in phase 4 pass the test suite, which was missing. Arguably, it implies that there possibly is some iteration on the test suite to match up.

I also added parentheses explaining the sanity check intention.

Reply:
Thanks for the parenthesized addition, but wanted to ask that we avoid the phrase "sanity check" to keep the language inclusive. Two ideas:

1. Replace the phrase with something like "smoke tests"
2. Get rid of the parentheses and make the explanation a proper part of the sentence, e.g. "... should pass on some implementation as a basic check that the test suite adequately covers the new features"

Reply:
🤯  Reworded.